virt/virt-2-4-release-notes.adoc

```diff
@@ -12,10 +12,18 @@ include::modules/virt-what-you-can-do-with-virt.adoc[leveloffset=+2]
 == New and changed features
 //Placeholder for new content.
 
-//CNV-4686 - Container-native Virtualization x" in the SVVP catalog listing
+//CNV-4686 - Container-native Virtualization x" in the SVVP catalog listing - Jira story was obsoleted.
 
 //CNV-4687 - Ensure platform passes Windows Server Virtualization Program with RHCOS workers.
 
+* {VirtProductName} is certified in Microsoft's Windows Server
+Virtualization Validation Program (SVVP) to run Windows Server workloads.
+** SVVP Certification applies to Intel and AMD CPUs.
+** SVVP Certificate Red Hat Enterprise Linux CoreOS 8 workers are named _Red Hat OpenShift
```

Comment:
`SVVP Certificate Red Hat Enterprise` sounds odd.

Maybe

`SVVP certified Red Hat Enterprise` … (at the end of the sentence) "in the Microsoft SVVP Ctalaog`
I'm adding this last part because it's unclear where the workers are named the way you describe it.

Reply:
Updated the content.